### DISCUSSION TITLE
Game grinds to a halt with 1.20.1 and Mobfarms

### DISCUSSION THREAD

**Discussion by Unrepentant-Atheist** [seymourimadeit/Big-Brain#59] [General]
Check this out: https://spark.lucko.me/x3kOqIpW4P

Have a simple mob farm, add EvilCraft spikes and pedestals to drain blood. With 8-10 mobs near these spike, the game grinds almost to a halt.

**Comment by seymourimadeit** [comment]
You can try disabling the Parkour ai in the config for now until I find a way to optimize it further

**Comment by Unrepentant-Atheist** [comment]
Which option is that?

I'm guessing: 	"Enable jumping ai" = true ?

**Comment by seymourimadeit** [comment]
Yeah

**Comment by seymourimadeit** [comment]
As an update on this I switched the algorithm for deciding the angle mobs jump at to what vanilla uses for breezes and goats rather than the copy and paste version I did years ago. I will keep this thread open but if there aren't any further reports in at most a year from now I'll probably consider this closed.
